mailbox/backup/src/test/java/org/apache/james/mailbox/backup/MailboxIdExtraFieldTest.java

```diff
@@ -0,0 +1,208 @@
+            assertThat(actual).isEqualTo(EMPTY_BYTE_ARRAY);
+        }
+
+        @Test
+        void getLocalFileDataDataShouldReturnValueInByteArray() {
+            byte[] actual = new MailboxIdExtraField(DEFAULT_MAILBOX_ID).getLocalFileDataData();
+            assertThat(actual).isEqualTo(DEFAULT_MAILBOX_ID_BYTE_ARRAY);
+        }
+    }
+
+    @Nested
+    class GetCentralDirectoryData {
+
+        @Test
+        void getCentralDirectoryDataShouldThrowWhenNoValue() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            assertThatThrownBy(() -> testee.getCentralDirectoryData())
+                .isInstanceOf(RuntimeException.class);
+        }
+
+        @Test
+        void getCentralDirectoryDataShouldReturnEmptyArrayWhenValueIsEmpty() {
+            byte[] actual = new MailboxIdExtraField(EMPTY).getCentralDirectoryData();
+            assertThat(actual).isEqualTo(EMPTY_BYTE_ARRAY);
+        }
+
+        @Test
+        void getCentralDirectoryDataShouldReturnValueInByteArray() {
+            byte[] actual = new MailboxIdExtraField(DEFAULT_MAILBOX_ID).getCentralDirectoryData();
+            assertThat(actual).isEqualTo(DEFAULT_MAILBOX_ID_BYTE_ARRAY);
+        }
+    }
+
+    @Nested
+    class ParseFromLocalFileData {
+
+        @Test
+        void parseFromLocalFileDataShouldParseWhenZero() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            testee.parseFromLocalFileData(EMPTY_BYTE_ARRAY, 0, 0);
+
+            assertThat(testee.getMailboxId().get())
+                .isEmpty();
+        }
+
+        @Test
+        void parseFromLocalFileDataShouldParseByteArray() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            testee.parseFromLocalFileData(DEFAULT_MAILBOX_ID_BYTE_ARRAY, 0, 16);
+
+            assertThat(testee.getMailboxId().get())
+                .isEqualTo(DEFAULT_MAILBOX_ID);
+        }
+
+        @Test
+        void parseFromLocalFileDataShouldHandleOffset() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            testee.parseFromLocalFileData(DEFAULT_MAILBOX_ID_BYTE_ARRAY, 2, 14);
+
+            assertThat(testee.getMailboxId().get())
+                .isEqualTo("3456789ABCDEF0");
+        }
+    }
+
+    @Nested
+    class ParseFromCentralDirectoryData {
+
+        @Test
+        void parseFromCentralDirectoryDataShouldParseWhenZero() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            testee.parseFromCentralDirectoryData(EMPTY_BYTE_ARRAY, 0, 0);
+
+            assertThat(testee.getMailboxId().get())
+                .isEmpty();
+        }
+
+        @Test
+        void parseFromCentralDirectoryDataShouldParseByteArray() {
+            MailboxIdExtraField testee = new MailboxIdExtraField();
+
+            testee.parseFromCentralDirectoryData(DEFAULT_MAILBOX_ID_BYTE_ARRAY, 0, 16);
+
+            assertThat(testee.getMailboxId().get())
```

Comment:
instead of .get().isEqualTo() you can write .contains() (and eveywhere in these tests)